docs/conf.py

```diff
@@ -36,7 +36,12 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["myst_parser", "sphinxcontrib.programoutput"]
+extensions = [
+    "myst_parser",
+    "sphinxcontrib.apidoc",
+    "sphinx.ext.intersphinx",
+    "sphinxcontrib.programoutput",
```

Comment:
These are imported and it's probably best to sort them following the same principles as normal import statements. I usually sort them as follows:
```suggestion
    # Stdlib extensions:
    "sphinx.ext.intersphinx",

    # Third-party extensions:
    "myst_parser",
    "sphinxcontrib.apidoc",
    "sphinxcontrib.programoutput",
```